esp/esp/program/modules/handlers/onsiteclasslist.py

```diff
@@ -149,8 +149,9 @@ def counts_status(self, request, tl, one, two, module, extra, prog):
     @needs_onsite
     def rooms_status(self, request, tl, one, two, module, extra, prog):
         resp = HttpResponse(mimetype='application/json')
-        data = ClassSection.objects.filter(status__gt=0, parent_class__status__gt=0, parent_class__parent_program=prog).select_related('resourceassignment__resource__name').values_list('id', 'resourceassignment__resource__name', 'resourceassignment__resource__num_students')
-        simplejson.dump(list(data), resp)
+        data = ClassSection.objects.filter(status__gt=0, parent_class__status__gt=0, parent_class__parent_program=prog).select_related('resourceassignment__resource__name').values_list('id', 'resourceassignment__resource__name', 'resourceassignment__resource__num_students', 'resourceassignment__resource__res_type__name')
+        data = [d for d in data if d[3] == 'Classroom']
```

Comment:
Does adding `resourceassignment__resource__res_type__name='Classroom'` to the filter not just work?  That will filter the returned rows in the query.  It might not be slow as-is, but inevitably these things get made more complicated over time, and in general it will be at least as fast in the DB, so I think it's just nicer to keep everything in the DB when possible